esx_service/vmdk_ops_test.py

```diff
@@ -479,7 +476,7 @@ def setUp(self):
         si = vmdk_ops.get_si()
         error, self.vm = create_vm(si=si, 
                                    vm_name=self.vm_name, 
-                                   datastore=self.datastore_name)
+                                   vm_path=self.datastore_name)
```

Comment:
Actually ```create_vm``` is internally prepares vm_path (vmHome) i.e. ```vm_path = datastore_name + vmDisplayName```.

So I am going to rename it with 'datastore_name' with more meaningful name to reduce the confusion.